Patches/WWII Soviet Faction/WWIISovietFaction_CE_Patch_RangedIndustrial.xml

```diff
@@ -0,0 +1,493 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+
```

Comment:
I'm generally going to assume these weapons have had their stats generated according to the Guns spreadsheet, and won't mark anything unless I think it's a anomaly. As always, try to use the existing CE melee tools for ranged weapons where possible.

Reply:
Yes, I forgot to mention that the weapons were balanced against [the FT copy](https://docs.google.com/spreadsheets/d/1Z9ZBnwRQWCKQm18O4I9ja4mFwLVChkNNfam2ZeQTTa0/edit#gid=884124215) of the latest CE Guns stats sheets.

All infantry firearm melee tools should already be standardized to standard CE melee tool values.